docs/development/justfile.md

```diff
@@ -0,0 +1,769 @@
+
+Aliases: `just dev`, `just serve`
+
+### Code Quality
+
+| Command | Description |
+|---------|-------------|
+| `just lint` | Run ESLint + Stylelint |
+| `just fix` | Auto-fix code style issues |
+| `just format-scss` | Format SCSS files with Prettier |
+
+Alias: `just l` for lint
+
+### Testing
+
+| Command | Description |
+|---------|-------------|
+| `just test` | Run Jest tests |
+| `just test --watch` | Run tests in watch mode |
+| `just ci-test` | Run tests with coverage (for CI) |
+| `just update-snapshots` | Update Jest snapshots |
+
+Alias: `just t` for test
+
+### Build
+
+| Command | Description |
+|---------|-------------|
+| `just build-nuxt` | Build Nuxt for local development |
+
+### Cleanup
+
+| Command | Description |
+|---------|-------------|
+| `just clean-nuxt` | Remove Nuxt build artifacts (.nuxt/, dist/) |
+| `just clean-all` | Remove all (build artifacts + node_modules) |
+
+### Passthrough
+
+| Command | Description |
+|---------|-------------|
+| `just yarn <args>` | Pass-through to yarn |
+| `just npm <args>` | Pass-through to npm |
+
+---
+
+## Tmux Development Sessions
+
+For a more integrated development experience, Baserow provides tmux-based sessions that open multiple panes with shells and logs.
+
+### Local Development with Tmux
+
+```bash
+just dev tmux
+```
+
+Starts a tmux session with:
+- Backend shell + dev server logs
+- Celery shell + worker logs
+- Database shell + logs
+- Redis logs
+- Frontend shell + dev server logs
+
+### Docker Development with Tmux
+
+```bash
+just dc-dev tmux
+```
+
+Starts a tmux session with Docker containers:
+- Backend container shell + logs
+- Celery container shell + logs (4 panes: shell, redbeat, worker, export worker)
+- Database shell + logs
+- Redis shell + logs
+- Frontend container shell + logs
+
+### Terminal Tabs (like dev.sh)
+
+```bash
+just dc-dev tabs
+or
+just dct
+```
+
+Opens terminal tabs for each service (similar to the old `dev.sh` behavior). Works with:
+- macOS Terminal.app
+- iTerm2
+- GNOME Terminal
+- Konsole
+
+---
+
+## Examples
+
+### Starting Fresh (Local Development)
+
+```bash
+cd backend
+just init
+# Edit .env.local as needed
+just run-dev-server
+```
+
+### Starting Fresh (Docker)
+
+```bash
+# From project root
+just dc-dev build --parallel
+just dc-dev up -d
+just dc-dev logs -f
+```
+
+### Running Tests
+
+```bash
+# All tests
+just test
+
+# Parallel execution
+just test -n=auto
+
+# Specific test file
+just test tests/baserow/core/test_models.py
+
+# With coverage
+just test-coverage
+```
+
+### Adding a Dependency
+
+```bash
+# Production dependency
+just deps-add httpx
+
+# Dev dependency
+just deps-add-dev pytest-benchmark
+```
+
+### Django Commands
+
+```bash
+# Make migrations
+just m makemigrations
+
+# Create superuser
+just m createsuperuser
+
+# Custom management command
+just m my_custom_command
+```
+
+### Celery Development
+
+The `run-dev-celery` command runs all Celery components together with colored output:
```

Comment:
Again, seems like instructions pretend to be relevant for the root folder context, so they don't work as expected.